radio/src/targets/common/arm/stm32/rtc_driver.cpp

```diff
@@ -67,27 +67,28 @@ void rtcInit()
   
   // Prevent lockup in case of 32kHz oscillator failure
   uint32_t i = 0;
-  while(RCC_GetFlagStatus(RCC_FLAG_LSERDY) == RESET)
-  {
-    if ( ++i > 1000000 )
+  while (RCC_GetFlagStatus(RCC_FLAG_LSERDY) == RESET) {
+    if (++i > 1000000)
       return;
   }
   
   RCC_RTCCLKConfig(RCC_RTCCLKSource_LSE);
   RCC_RTCCLKCmd(ENABLE);
   RTC_WaitForSynchro();
 
+#if !defined(BOOT)
   // RTC time base = LSE / ((AsynchPrediv+1) * (SynchPrediv+1)) = 1 Hz*/
   RTC_InitStruct.RTC_HourFormat = RTC_HourFormat_24;
   RTC_InitStruct.RTC_AsynchPrediv = 127;
   RTC_InitStruct.RTC_SynchPrediv = 255;
   RTC_Init(&RTC_InitStruct);
-  
+
   struct gtm utm;
   rtcGetTime(&utm);
   g_rtcTime = gmktime(&utm);
+#endif
 
-#if defined(RAMBACKUP)
+#if defined(RTC_BACKUP_RAM) && !defined(BOOT)
   RCC_AHB1PeriphClockCmd(RCC_AHB1Periph_BKPSRAM, ENABLE);
   PWR_BackupRegulatorCmd(ENABLE);
```

Comment:
The manual says that `PWR_GetFlagStatus(PWR_FLAG_BRR)` should be checked after the regulator is turned on to make sure the regulator is ready. Is this something we should care about?

Reply:
well, we have not need any issue since day 1 on this, I propose to keep it unchanged